draft-ietf-rats-msg-wrap.md

```diff
@@ -637,6 +638,13 @@ Client-authenticated TLS may be used to bind a Collection CMW of Evidence messag
 However, the client key used with TLS should not be that of the end-user or owner of the device.
 Instead, it should be attestation-oriented key material from the device or the attester manufacturer.
 
+## CMW in X.509 {#seccons-x509}
+
+Publishing attestation information in a public certificate raises privacy and, in some cases, security concerns.
```

Comment:
```suggestion
Publishing attestation information (Attestation Results) in a public certificate raises privacy and, in some cases, security concerns.
```

Reply:
Ohh. Good point. I was thinking of evidence more than attestation results.
You need to be really careful putting evidence into a X.509 cert (and I'm sure some people will want to, still a little bit careful but probably less so with ARs.